actiontext/test/unit/attachable_test.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ActionText::AttachableTest < ActiveSupport::TestCase
+  test "as_json is a hash when the attachable is persisted" do
+    attachable = ActiveStorage::Blob.create_after_unfurling!(io: StringIO.new("test"), filename: "test.txt")
+
+    assert_kind_of Hash, attachable.as_json
```

Comment:
Why we don't assert that the hash should be?

Reply:
@rafaelfranca thanks for your comment. 
I asserted what the hash should be, squashed my commits and rebased on main.